src/main/scala/com/johnsnowlabs/ml/util/LoadExternalModel.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017-2022 John Snow Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.johnsnowlabs.ml.util
+
+import com.johnsnowlabs.nlp.util.io.{ExternalResource, ReadAs, ResourceHelper}
+
+import java.io.File
+
+object LoadExternalModel {
+
+  def modelSanityCheck(modelPath: String): String = {
```

Comment:
I moved the check for vocab.txt here as well (and potentially other assets). Is this something that we would want?

https://github.com/JohnSnowLabs/spark-nlp/pull/12931/files#diff-13282f334c1c1e8d9a1253a31cce7d87da16659719b1b606f251e6688e420a75R23